requirements-test.txt

```diff
@@ -1,4 +1,4 @@
-coverage==4.5.1
-pytest==3.0.7
-pytest-cov==2.5.1
-tox<3.0
+coverage==6.2
+pytest==7.1.1
+pytest-cov==3.0.0
+tox==3.24.5
```

Comment:
maybe we should less restrictive versioning?

Reply:
Definitely. Restricted to major version number in new commit.